config.js.template

```diff
@@ -2,6 +2,7 @@ module.exports = {
   // HTTP API Settings
   http_bind_address: 'localhost',
   http_port: 8000,
+  http_api_key: null,
```

Comment:
Should we add a comment here saying it's optional?

Reply:
Fixed. Thanks!